source/docs/prismic/routable-types.md

```diff
@@ -0,0 +1,231 @@
+---
+id: routable-types
+title: Prismic Routable Types
+---
+
+To make a custom prismic data type routable these steps are required:
+
+- [Create a field to represent the Url of the custom type](#create-a-field-to-represent-the-url-of-the-custom-type)
+
```

Comment:
don't put an empty line between bullet points otherwise it gets renders several list instead of one (same for other lists in the page)